README.md

```diff
@@ -55,7 +55,37 @@ See [INSTALL.md](INSTALL.md).
 
 ## How to Cite
 
-When using this work, please cite an appropriate version on Zenodo: https://doi.org/10.5281/zenodo.8155057
+Please cite the benchmark collection by citing:
+1. the PEtab paper, which first introduced this collection: https://doi.org/10.1371/journal.pcbi.1008646
+2. the Zenodo repository, where versions of this collection are deposited (doi:10.5281/zenodo.8155057)
```

Comment:
So people can use the version-specific Zenodo DOI if they choose, instead of the generic DOI in the example bibtex below
```suggestion
2. the Zenodo repository, where versions of this collection are deposited (https://doi.org/10.5281/zenodo.8155057)
```